tests/modules/nf-core/purgedups/histplot/nextflow.config

```diff
@@ -0,0 +1,14 @@
+process {
+
+    publishDir = { "${params.outdir}/${task.process.tokenize(':')[-1].tokenize('_')[0].toLowerCase()}" }
+
+    withName: 'PURGEDUPS_(PBCSTAT|CALCUTS)' {
+        publishDir = [ enabled: false ]
+    }
+
+}
+
+// The tool is not in the environment using conda, but is using mamba or container platform.
+// conda.useMamba = true
```

Comment:
Is this supposed to be here? As the tests suggest it is working in conda.

Reply:
The CI tests use the mamba libsolver so conda builds the environment correctly. When it's using it's classical solver, conda chooses the previous build (not latest) meaning certain scripts and executables are not in the conda environment. However the libmamba should be the default solver in the next release of conda so hopefully that should take care of the issue in future. I'll get rid of the comment.